ib_manifest_util/util.py

```diff
@@ -75,18 +75,23 @@ def run_subprocess(command: str):
         sys.stdout.write(line.decode("utf-8"))
 
 
-def download_file(url: str):
+def download_file(url: str) -> str:
```

Comment:
I agree with @rycrow here, we should allow for a single url or a list of str to download multiple files.

Reply:
I can add it to this PR.

Reply:
I added the changes. @rycrow @kcpevey